### PR TITLE
reading time in meta

### PR DESCRIPTION
**Pull request by fuentesloic**

### CODE REVIEW

**Comment by fuentesloic** [comment]
Just a drop to run my mind on this way :) 
So the calcul method is not good yet ! 

Python fan: @Ydrasil  & @vinyll let me see you super power :)

**Review by vinyll — APPROVED** [review]

**Comment by vinyll** [comment]
Round cf: http://jinja.pocoo.org/docs/2.9/templates/#round

Ex: `{{ (article.content|length / 6 / 250)|int }}`

**Comment by fuentesloic** [comment]
for exemple = `{{ (document.content|lenght / 6 / 300) | int }}`

File: templates/blog/article.html [hunk 1]
```diff
@@ -26,6 +26,7 @@
       <a href={{ url_for_user(article.author) }}>{{ article.author.username }}</a>
       <p>{{ article.author.about|htmlnewline }}</p>
       <time datetime={{ article.creation_date }} pubdate>{{ article.creation_date|datetime }}</time>
+      <p class="reading-time">Reading time: <span>{{ (article.content|length / 6 / 250)|int }}</span> min</p>
```
Comment (vinyll) [inline templates/blog/article.html]: remove quotes

File: templates/blog/macros.html [hunk 2]
```diff
@@ -10,6 +10,7 @@
       <div>
         <a href={{ url_for_user(article.author) }}>{{ article.author.username }}</a>
         <time datetime={{ article.creation_date }} pubdate>Published on {{ article.creation_date|datetime }}</time>
+        <p class="reading-time">Reading time: <span>{{ (article.content|length / 6 / 250)|int }}</span> min</p>
```
Comment (vinyll) [inline templates/blog/macros.html]: remove quotes

**Review by vinyll — CHANGES_REQUESTED** [review]

**Comment by vinyll** [comment]
We could copy Medium's style:
screen shot 2017-06-25 at 4 32 54 pm

**Pull request merged**